sci-mathematics/octave/octave-7.1.0.ebuild

```diff
@@ -0,0 +1,213 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic fortran-2 java-pkg-opt-2 pax-utils qmake-utils toolchain-funcs xdg-utils
+
+DESCRIPTION="High-level interactive language for numerical computations"
+HOMEPAGE="https://www.gnu.org/software/octave/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+LICENSE="GPL-3"
+
+SLOT="0/${PV}"
+IUSE="curl doc fftw +glpk gnuplot gui hdf5 java opengl
+	portaudio postscript +qhull +qrupdate readline sndfile +sparse
+	ssl static-libs sundials X zlib"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
+
+# Although it is listed in INSTALL.OCTAVE as a build tool, Octave runs
+# "makeinfo" from sys-apps/texinfo at runtime to convert its texinfo
+# documentation to text (see scripts/help/help.m).
+#
+# (un)zip isn't mentioned, but there's a test that uses it (bug 775254).
+#
+RDEPEND="
+	app-arch/bzip2
+	app-arch/unzip
+	app-arch/zip
+	app-text/ghostscript-gpl
+	sys-apps/texinfo
+	dev-libs/libpcre:3=
+	sys-libs/ncurses:0=
+	sys-libs/zlib
+	virtual/blas
+	virtual/lapack
+	curl? ( net-misc/curl:0= )
+	fftw? ( sci-libs/fftw:3.0= )
+	glpk? ( sci-mathematics/glpk:0= )
+	gnuplot? ( sci-visualization/gnuplot )
+	gui? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qthelp:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtopengl:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtwidgets:5
+		x11-libs/qscintilla:=
+	)
+	hdf5? ( sci-libs/hdf5:0= )
+	java? ( >=virtual/jre-1.6.0:* )
```

Comment:
java? ( >=virtual/jre-1.8:* )

Same for 6.4.0-r2
See https://wiki.gentoo.org/wiki/Java_Developer_Guide#Dependencies